package.json

```diff
@@ -46,22 +46,21 @@
     "test": "jest --watch"
   },
   "dependencies": {
-    "tslib": "^2.4.0"
+    "tslib": "^2.6.2"
   },
   "devDependencies": {
     "@babel/core": "^7.17.10",
-    "@rollup/plugin-commonjs": "^22.0.0",
-    "@rollup/plugin-node-resolve": "^13.2.1",
+    "@rollup/plugin-commonjs": "^25.0.7",
+    "@rollup/plugin-node-resolve": "^15.2.3",
+    "@rollup/plugin-terser": "^0.4.4",
     "@types/jest": "^27.4.1",
     "@types/node": "^17.0.30",
     "babel-jest": "^28.0.3",
-    "camelcase": "^6.3.0",
+    "camelcase": "^8.0.0",
     "jest": "^28.0.3",
     "jest-environment-jsdom": "^28.0.2",
     "npm-run-all": "^4.1.5",
-    "rollup": "^2.70.2",
-    "rollup-plugin-sourcemaps": "^0.6.3",
-    "rollup-plugin-terser": "^7.0.2",
```

Comment:
I've replaced `rollup-plugin-terser` with `@rollup/plugin-terser`, which is the official Rollup plugin.